ftrr/src/main/java/framgia/co/edu/ftrr/service/impl/InterviewServiceImpl.java

```diff
@@ -111,4 +124,17 @@ else if (division.isHd())
     private Boolean checkDateTime(Date startTimeToCheck, Date endTimeToCheck, Date startTimeOrigin, Date endTimeOrigin) {
         return startTimeToCheck.after(endTimeOrigin) || startTimeOrigin.after(endTimeToCheck);
     }
+
+    private Boolean checkRequest(Request request) {
```

Comment:
`checkRequest`
chưa rõ là em muốn check cái gì của request, nếu muốn biết, chị phải đọc code bên trong
=> tạo tên method làm sao để ng ta đọc cái tên method là bik trong đó xử lý những gì (readable coding - sách đó - mặc dù tìm tên để đặt là 1 vấn đề nan giải)

`is_completed_results_interview_for_request ` thì sao em
chị thêm `_` cho dễ đọc thôi, chứ em thay `_` bằng chữ in hoa hì